### ISSUE TITLE
[Quest][Outland] Turning Point / Deathblow to the Legion

### ISSUE THREAD

**Issue by riddermark** [dalaranwow/dalaran-wow#855]
Turning Point
http://www.wowhead.com/quest=10507 (Scryers version)

Deathblow to the Legion
http://www.wowhead.com/quest=10409 (Aldor Version)  https://github.com/dalaranwow/dalaran-wow/issues/819?source=cc

Problem atm is the boss is not attackable (invalid target) and the 2nd item from the list below cant be used on him.

You get 2 items:
http://www.wowhead.com/item=29796#comments (to teleport there)
http://www.wowhead.com/item=30259 (to weaken the boss as follows:
- used to debuff the boss, making his damage dealt reduced by 75% and Damage taken increased by 300%.

Boss and his abilities:
http://www.wowhead.com/npc=20132#abilities

There is an Aldor questline that leads to this and this NPC:
http://www.wowhead.com/npc=20794#comments

He is skippable and there's no need to attack him if you are not on that Aldor quest or a Scryer. I believe he is trigged by dialogue (again aldor on quest only).

After killing this guy you get the fight with Socrethar. This NPC should tank it for you:
http://www.wowhead.com/npc=18538/ishanah#starts
- Now this part is the event thing that happens so let me just re-tell from all the comments I've managed to pull together:

3 Aldor NPCs spawn, and you tell one of them that "I'm ready."
Ive figured 1 is this - Ishanah NPC and number 2 is:
http://www.wowhead.com/npc=18537/adyen-the-lightwarden#comments

The 3rd NPC is still a mystery

NPCs will proceed to Socrethar. They will start talking, after a while Kaylaan will spawn and Socrethar will cast a spell on him and make him fight us.
Kaylaan will cast "Heal" when he is low on HP, can be interrupted.

Soon after, Ishanah will spawn and start talking again. Ishanah and the other 2 Aldor NPCs will attack Socrethar, and the main point is, Ishanah will be tanking Socrethar. Ishanah is an uber tank and she can heal herself.

More info:
1. Go to the teleporter at 36,18 and right click your stone,
2. When you are teleported to Socrethar's seat, several Aldor will appear. Tell Adyen you're ready or you will have to fight this bad boy alone; not advisable.
3. The Aldor npc's will lead you to Socrethar. You'll kill a few mobs before you reach him, and then there will be some drama before the final fight. I don't want to spoil it for you, but know that the Aldor npc you'll have to fight doesn't have to be killed. That's a good thing, because he keeps healing himself back to full health.

AS for the exat dialogues that happen - I cannot find any source

Wowiki entry for the Aldor quest:
http://www.wowwiki.com/Quest:Deathblow_to_the_Legion
For the scryer version:
http://www.wowwiki.com/Quest:Turning_Point

Only crappy video that doesnt show the RP event:
http://www.youtube.com/watch?v=0PlDZ20yNZc

Sorry for the crappy report but I can't really find much more.

**Comment by ashido96** [comment]
NPC is still un-attackable, nor can the items be used on him.

**Comment by Cone182** [comment]
still bugged

**Comment by Larax323** [comment]
Still bugged :( jesus 2 and a half years problem exists....no one lifts a finger

**Comment by Zibathy** [comment]
Still bugged on Kirin Tor.

**Comment by Rushor** [comment]
DB/Quest: Turning Point / Deathblow to the Legion - Socrethar now rewards the questcredit and is attackable

* closes https://github.com/dalaranwow/dalaran-wow/issues/855
* this does not include the questscript, but at least the quest is now completable for player
* so please @tester enable this fix to get applied on live and open a new issue with detailed information on the missing RP Event. Thank you.
* commit: 6febe592fac895ee4f980ddc9c53688e28351564

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by joki8** [comment]
Socrethar is atackable but no complete quest when you kill him. No aldor still spawn and kaylaan still do nothing.

**Comment by tenebray** [comment]
Socrethar is attackable but quest does not complete on kill. For Aldor version at least
